lib/src/mock_user.dart

```diff
@@ -73,8 +75,36 @@ class MockUser with EquatableMixin implements User {
   String? get refreshToken => _refreshToken;
 
   @override
-  Future<String> getIdToken([bool forceRefresh = false]) async {
-    return Future.value('fake_token');
+  Future<String> getIdToken([bool forceRefresh = false]) {
+    var payload = {
+      'name': displayName,
+      'picture': photoURL,
+      'iss': 'fake_iss',
+      'aud': 'fake_aud',
+      'auth_time': 1655946582,
```

Comment:
What date is this? Would you mind instantiating a human readable date, then convert it to whatever format this is?

Reply:
I just copy these dates from my real project firebase user item.  I think I can make MockUser's constructor accept some human-readable dates and generate these iat and exp.